### ISSUE TITLE
 permutation Importance warning scikit 0.25

### ISSUE THREAD

**Issue by ThomasWolf0701** [eli5-org/eli5#12]
When running permutation Importance I get the following warning:
 
C:\Users\Thomas Wolf\anaconda3\lib\site-packages\sklearn\utils\validation.py:70: FutureWarning: Pass classifier=False as keyword args. From version 0.25 passing these as positional arguments will result in an error
  FutureWarning)
C:\Users\Thomas Wolf\anaconda3\lib\site-packages\sklearn\utils\validation.py:70: FutureWarning: Pass classifier=False as keyword args. From version 0.25 passing these as positional arguments will result in an error
  FutureWarning)

**Comment by zeromh** [comment]
Same! Would love to see this fixed before it breaks.

**Comment by johncant** [comment]
Looks like this has been fixed in the latest `master`

**Comment by ThomasWolf0701** [comment]
> Looks like this has been fixed in the latest `master`

Still looks the same as before to me. The pypi version is still the old one too.

**Comment by ThomasWolf0701** [comment]
cv = check_cv(self.cv, y, classifier=is_classifier(self.estimator))  seems to be the culprit in the permutation_importance.py function.

**Comment by ThomasWolf0701** [comment]
cv = check_cv(self.cv, y, is_classifier(self.estimator))
TypeError: check_cv() takes from 0 to 2 positional arguments but 3 were given

Is the error message with scikit-learn 1.0.2

**Comment by lopuhin** [comment]
This should be fixed in #48  , we're planning a PyPI release soon, closing this one as there are other issues open about sklearn compatibility, e.g. https://github.com/eli5-org/eli5/issues/42
